Server/mods/deathmatch/logic/packets/CEntityAddPacket.cpp

```diff
@@ -1025,6 +1025,15 @@ bool CEntityAddPacket::Write(NetBitStreamInterface& BitStream) const
                                 vertex.data.vecPosition = *iter;
                                 BitStream.Write(&vertex);
                             }
+
+                            if (BitStream.Can(eBitStreamVersion::SetPolygonHeight))
```

Comment:
Better clarification.
```suggestion
                            if (BitStream.Can(eBitStreamVersion::SetColPolygonHeight))
```